polkadot/runtime/common/src/identity_migrator.rs

```diff
@@ -0,0 +1,305 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
```

Comment:
can we stop adding more code to polkadot-runtime-common https://github.com/paritytech/polkadot-sdk/issues/2186